lib/java/ydoc-server/src/main/java/org/enso/ydoc/server/Main.java

```diff
@@ -25,21 +28,22 @@ public static void main(String[] args) throws Exception {
       var then = System.currentTimeMillis();
```

Comment:
- Is this `public static void main` still useful for anything?
- Shouldn't it be removed?